### PR TITLE
[CUBRIDQA-12] fix unstable test cases, updated expected answer.

### PR DESCRIPTION
**Pull request by jinmeihua**
**_08_current_date_function/_02_functions/cases/_13_datediff.sql**

- https://github.com/CUBRID/cubrid-testcases/pull/25/commits/d5ef0957c63ef7ee18ed14a3c8cef0c04c78aed5
When the datetime is '2016-07-01 02:37:00 KST', 'hour(@x4)-hour(@x1)' is '3'. So add '3' to expected answer.

**_08_current_date_function/_02_functions/cases/_19_round.sql**
- https://github.com/CUBRID/cubrid-testcases/pull/25/commits/322c7f6cd6ee431ebdc1f216abe853b7f18a26bb
It doesn't make sense.
- https://github.com/CUBRID/cubrid-testcases/pull/25/commits/8f78f49b39810a5b91937f1483f7c8ed2a2ee36d
The test case '_08_current_date_function/_02_functions/cases/_19_round.sql' is add for current_date testing, but it is no much meaning. So I think we'd better delete this test case.

### CODE REVIEW

File: sql/_27_banana_qa/issue_5765_timezone_support/_08_current_date_function/_02_functions/cases/_19_round.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 --+ holdcas on;
-set timezone '-01:00';
+set timezone '+01:00';
```
Comment (jinmeihua) [inline sql/_27_banana_qa/issue_5765_timezone_support/_08_current_date_function/_02_functions/cases/_19_round.sql]: It doesn't make sense, I will consider other ways.

**Comment by slamdunkorchid** [comment]
Please commit it.

**Comment by fanzaiqiang** [comment]
@jinmeihua 

It's better keep the case. 

We may consider set the later timezone with adjacent value compare to the first one.

**Comment by jinmeihua** [comment]
@slamdunkorchid , @fanzaiqiang 
I opened a new PR(https://github.com/CUBRID/cubrid-testcases/pull/26) to do above changes.
When I commit the https://github.com/CUBRID/cubrid-testcases/pull/25/commits/322c7f6cd6ee431ebdc1f216abe853b7f18a26bb , the system date was set as 2016-12-31. So, I want to close this PR.

**Pull request closed without merging**